### ISSUE TITLE
Admin UI: Unify focus styling across links, tabs, buttons, etc

### ISSUE THREAD

**Issue by ryelle** [Automattic/jetpack#6531]
Focus styles are useful for users who use the keyboard to navigate through a site. In the current UI, there are focus styles for almost everything, but they aren't consistent (or even good, in some places). See below, there are 3 different kinds of style for focus on just this one section of page. And calypso form components use yet another style. 

![dashboard](https://cldup.com/6ueCsLiJ8C.png)

![settings](https://cldup.com/tvEsWKaZJK.png)

The first two in the first screenshot are OK, but the last ("Weeks"), is not different enough from the currently-selected item; additionally there is no focus indication when landing on the currently selected item.

The setting screens are otherwise good, the form styling from calypso is carried over.
